pkg/sinks/http/http.go

```diff
@@ -82,19 +85,20 @@ func NewSink(log logger.Underlying, registry go_metrics.Registry, sink string) (
 }
 
 func (s *HttpSink) Init(ctx context.Context, format formats.Format, compression kt.Compression, fmtr formats.Formatter) error {
-	s.TargetUrl = *TargetUrl
+	for _, url := range strings.Split(*TargetUrl, ",") {
+		s.targetUrls = append(s.targetUrls, url)
+		s.Infof("Exporting HTTP to %s", url)
+	}
 
 	s.tr = &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: *InsecureSkipVerify},
 	}
 	s.client = &http.Client{Transport: s.tr}
 
 	if compression == kt.CompressionGzip {
 		s.headers["Content-Encoding"] = "GZIP"
 	}
 
-	s.Infof("Exporting to Http at %s", s.TargetUrl)
-
 	return nil
 }
 
```

Comment:
Since we could potentially have several targets should we wrap the `sendHttp` context with a `WithTimeout` and specify a max to prevent dead or slow targets from stacking up goroutines?  That way we could also log it to know when a target is behaving badly?

Reply:
Definitely.

Reply:
I think I did the timeout right but I always get confused with this thing.

Reply:
:+1: